modules/mobile/docs/openapi.yaml

```diff
@@ -1284,6 +1483,31 @@ components:
         financialInstitutionRoutingNumber:
           type: string
           example: "021000021"
+    PushRegistration:
+      required:
+        - deviceToken
+        - osName
+        - osVersion
+        - deviceName
+      properties:
+        deviceToken:
+          type: string
+          example: "740f4707bebcf74f9b7c25d48e3358945f6aa01da5ddb387462c7eaf61bb78ad"
+        osName:
+          type: string
+          enum:
+            - ios
+            - android 
+        osVersion:
+          type: string
+          example: "12.1"
+        deviceName:
+          type: string
+          example: "Jeff's iPhone"
+    PushPrefs:
+      properties:
+        appointment_reminders:
+          type: boolean
```

Comment:
We only will support appointment_reminders to start, but what's the best way to handle a growing list of notification types?

Reply:
Depends how the FE wants to handle it, if the FE 'knows' what preference keys will be available then this works. If it's meant to be truly dynamic then it could be an array of objects with maybe `[{ "name": "appointment_reminders", "enabled": true}]`

Reply:
I think truly dynamic will server the API best in the long run. Changes made.